src/views/Workflow.vue

```diff
@@ -117,15 +119,16 @@ export default {
      * @type {CylcTree}
      */
     tree: new CylcTree(),
-    table: [],
+    table: {},
     isLoading: true,
     // the widgets added to the view
     /**
      * @type {
      *   Object.<string, string>
      * }
      */
-    widgets: {}
+    widgets: {},
+    deltasCallbacks: []
```

Comment:
I'm not sure if this is the best way... it's not being cleaned up when we remove widgets... I think it's the last bit before it's ready for review.